gems/manageiq_ansible_tower/manageiq_ansible_tower.gemspec

```diff
@@ -0,0 +1,36 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'manageiq_ansible_tower/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = "manageiq_ansible_tower"
+  spec.version       = ManageiqAnsibleTower::VERSION
+  spec.authors       = ["Brandon Dunne"]
+  spec.email         = ["bdunne@redhat.com"]
+
+  spec.summary       = %q{Ansible Tower Provider for ManageIQ}
+  spec.description   = %q{Ansible Tower Provider for ManageIQ}
```

Comment:
Are we going to use enough of their API to warrant taking on maintaining a full API wrapper? Is the API complex enough that it *needs* a wrapper?